src/index.html

```diff
@@ -11,12 +11,95 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute of the `img` tag should describe the image content. 'logo' is not descriptive enough. It would be better if you provide more details about the logo. For example, if it's a company logo, you can include the company name and what's on the logo. (HTML Content)